definitions/json/AuthorisationCaseState.json

```diff
@@ -14,12 +20,12 @@
   {
     "CaseTypeID": "ET_EnglandWales",
     "CaseStateID": "Submitted",
-    "UserRole": "caseworker-employment-api",
-    "CRUD": "CRUD"
+    "UserRole": "caseworker-employment-etjudge-englandwales",
```

Comment:
Were these supposed to be changed or was it by accident?

Reply:
It is how GitHub shows that somehow api role has been removed (if you see the next row has been modified as well with api role and submitted state). If you open this branch in IntelliJ, it's there.

  {
    "CaseTypeID": "ET_EnglandWales",
    "CaseStateID": "Submitted",
    "UserRole": "caseworker-employment-api",
    "CRUD": "CRUD"
  },

Reply:
Ah yes, my bad

Reply:
Sorry that's cleared one thing but how about the 'Draft' state?